book.json

```diff
@@ -3,7 +3,7 @@
   "root": "./docs",
   "structure": {
     "readme": "Introduction.md",
-    "summary": "Readme.md"
+    "summary": "Summary.md"
```

Comment:
I found it confusing that this file was named "Readme.md" then passed to the "summary" field when there was another "readme" field available.

Though if I'm to be perfectly honest, the real motivation behind this was that at 1am this morning I was stupidly wandering around the repo Readme.md file trying to figure out where this config list was located 😆

Reply:
Very fair, I think the reason I did that was to make it show up as the readme on GitHub, but I'm not tied to it. Thank you!